### ISSUE TITLE
Cross-vertical-pane corruption in Gnome Terminal

### ISSUE THREAD

**Issue by kamalmarhubi** [tmux/tmux#1745]
### Issue description

Output in a vertical pane on the right interferes and corrupts the contents of the left pane. This reproduces under Gnome Terminal 3.32.2 (VTE 0.56.3) but not under xterm 334.

Steps to reproduce:
- start tmux with no config
- produce any output, eg `seq 10`
- create a new vertical pane

At this point, some output from the `seq 10` command disappears.

More corruption is visible with further interaction, eg running `man` in the right-hand pane produces lots of garbled `man` output on the left.

### Required information

Please provide the following information:

#### tmux version
tmux built at current HEAD (54da4934767a1868c6ac0efb5d1ce6edd2b24f5d):
```
$ ./tmux -V
tmux next-3.0
$ git rev-parse HEAD
54da4934767a1868c6ac0efb5d1ce6edd2b24f5d
```
#### Platform:
```
$ uname -sp
Linux x86_64
```
`$TERM` inside tmux:
```
$ echo $TERM
screen
```
`$TERM` outside tmux:
```
$ echo $TERM
xterm-256color
```
(Also reproduces with `TERM=vte`.)

#### Logs
Logs from following the sequence described at top, including running `man ./tmux.1`:

[tmux-client-18434.log](https://github.com/tmux/tmux/files/3194736/tmux-client-18434.log)
[tmux-out-18436.log](https://github.com/tmux/tmux/files/3194737/tmux-out-18436.log)
[tmux-server-18436.log](https://github.com/tmux/tmux/files/3194738/tmux-server-18436.log)

**Comment by nicm** [comment]
Can't reproduce with GNOME Terminal 3.30.2, VTE 0.54.2.

I don't see anything obvious in your logs. The fact that this doesn't happen on xterm means it is probably a VTE issue.

Can you reproduce on an 80x24 terminal please and send me the logs from that?

**Comment by kamalmarhubi** [comment]
> The fact that this doesn't happen on xterm means it is probably a VTE issue.

As a tried to minimise the steps to reproduce I started to think this might be the case, but not sure how to verify it; I'm finding it a bit difficult to work around Gnome Terminal's client-server architecture. Do you have any tips?

Logs from 80x24 showing the behaviour, otherwise same config:
[tmux-client-28196.log](https://github.com/tmux/tmux/files/3195583/tmux-client-28196.log)
[tmux-out-28198.log](https://github.com/tmux/tmux/files/3195584/tmux-out-28198.log)
[tmux-server-28198.log](https://github.com/tmux/tmux/files/3195585/tmux-server-28198.log)

**Comment by nicm** [comment]
Can you open an issue with VTE and see if they can reproduce? I can't reproduce even using exactly what tmux is sending from your logs, but my gnome-terminal and VTE versions are older and building it is too fiddly.

**Comment by kamalmarhubi** [comment]
After some further experimentation, I'm starting to think this is an interaction with [podman], as the repro environment was inside a [toolbox] container. It does not reproduce on the same system directly on the "host". I was sure I saw this in both "host" and "container" environemnts, but apparently I was wrong.

Closing this out as I no longer think it's actionable; will update if / when I resolve the issue to provide some breadcrumbs.

[podman]: https://podman.io/
[toolbox]: https://github.com/debarshiray/toolbox
